iambic/config/wizard.py

```diff
@@ -360,12 +360,19 @@ def __init__(self, repo_dir: str, is_more_options: bool = False):
 
     @property
     def has_cf_permissions(self):
-        if self._has_cf_permissions is None:
+        if self._has_cf_permissions is None or self._has_cf_permissions is False:
```

Comment:
Can't we just do `if not self._has_cf_permissions`?

Reply:
I guess that the value of _has_cf_permission should always 

```
self._has_cf_permissions = (
    "member.org.stacksets.cloudformation.amazonaws.com"
    in [
        p["ServicePrincipal"]
        for p in org_client.list_aws_service_access_for_organization()[
            "EnabledServicePrincipals"
        ]
    ]
)
```

Reply:
I mean, the return value does not depend on `_has_cf_permissions`

when _has_cf_permissions is filled:
- during set_boto3_session
- during setup_aws_configuration
  - I guess is being set always on each if-branch
  - in the end, it calls set_boto3_session

scenario:
- when you select aws on setup

if it is always being call, should it be instanciated as property?

Reply:
solved